client/src/client_proxy.rs

```diff
@@ -367,6 +377,28 @@ impl ClientProxy {
         })
     }
 
+    /// Prepare a transfer transaction: return the unsigned raw transaction
+    pub fn prepare_transfer_coins_int(
+        &mut self,
+        sender_address: AccountAddress,
+        sender_sequence_number: u64,
+        receiver_address: &AccountAddress,
+        num_coins: u64,
+        gas_unit_price: Option<u64>,
+        max_gas_amount: Option<u64>,
+    ) -> Result<RawTransaction> {
+        let program = vm_genesis::encode_transfer_program(&receiver_address, num_coins);
+        let unsigned_tx = self.create_unsigned_transaction(
+            program,
+            sender_address,
+            sender_sequence_number,
+            max_gas_amount, /* max_gas_amount */
+            gas_unit_price, /* gas_unit_price */
```

Comment:
You don't need the comments on 395 and 396 since they are already named identical to the comment

Reply:
deleted the comments